#/usr/bin/env python
APP_HOST = 'info3103.cs.unb.ca'
APP_PORT = <!INSERT_PORT_HERE!>
APP_DEBUG = True

DB_HOST = 'localhost'
DB_USER = <!INSERT_USER_HERE!>
DB_PASSWD = <!INSERT_PASSWORD_HERE!>
DB_DATABASE = DB_USER

SECRET_KEY = <!ENTER_LDAP_SECRET_KEY_HERE!>

LDAP_HOST = 'ldap-student.cs.unb.ca'
